src/index.html

```diff
@@ -9,6 +9,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="container">
+        <nav class="container_1">
+          <img src="https://github.com/yaroslav-furhala/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
+          class="link_box"
+          alt="moyo">
```

Comment:
```suggestion
          <img 
            src="https://github.com/yaroslav-furhala/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
            class="link_box"
            alt="moyo"
          >
```

Reply:
also change structure as in previous comment